### PR TITLE
Update Breaker show link rule

### PR DESCRIPTION
**Pull request by sinchang**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,6 +26,7 @@ In this context, a `slug` is a string with a malleable, arbitrary value. For som
 
 ### Non-deterministic Platform Show Links
 * Amazon Music: `https://music.amazon.com/podcasts/${uniquePlatformID}`
+* Breaker: `https://breaker.audio/${slug}`
```
Comment (nathangathright) [inline README.md]:
```suggestion
* Breaker: `https://breaker.audio/${uniquePlatformID}`
```

**Review by nathangathright — CHANGES_REQUESTED** [review]
For the purposes of this document, `slug` is defined as a string with a malleable, arbitrary value. `uniquePlatformID` is the correct term.

File: README.md [hunk 2]
```diff
@@ -43,7 +43,7 @@ In this context, a `slug` is a string with a malleable, arbitrary value. For som
 
 ### Non-deterministic Platform Episode Links
 * Apple Podcasts: `https://podcasts.apple.com/podcast/id${appleID}?i=${uniqueEpisodeID}`
-* Breaker: `https://breaker.audio/${uniquePlatformID}/e/${uniqueEpisodeID}`
+* Breaker: `https://breaker.audio/${slug}/e/${uniqueEpisodeID}`
```
Comment (nathangathright) [inline README.md]:
```suggestion
* Breaker: `https://breaker.audio/${uniquePlatformID}/e/${uniqueEpisodeID}`
```

**Comment by nathangathright** [comment]
I've spoken with Greg Wasserman at Maple Media, owner of Breaker, and he's provided this statement:
> Yes we are aware and sorry for the confusion, we are in the process of taking that page down, and if you have a show you want to submit or update on Breaker, please contact support and include the RSS feed. Support can be reached at feedback@breaker.audio

**Comment by sinchang** [comment]
@nathangathright thanks for the clarification.

**Pull request merged**